### PR TITLE
modified rescale description

### PR DESCRIPTION
**Pull request by MasanoriTakaoka**
"rescaling factor" in original English text doesn't relate image size. Actually, rescale parameter affects image data like RGB data.
rescale パラメータはサイズではなく、画素値を変換するものなのなので、誤解を招かないように「画像データ」としたほうが良いかと思います。

### CODE REVIEW

**Comment by nzw0301** [comment]
Thank you for your PR. I think that `画像データの変更の係数` is an unclear description of this parameter. `標準化係数` may be better to explain the role of this parameter.

**Comment by MasanoriTakaoka** [comment]
Thanks for your comment. In my understanding, normalization `正規化` and standardization `標準化` are different meaning. In this case, I think normalization`正規化` is preferable. So how about `画像データの正規化係数`?

**Comment by nzw0301** [comment]
Thank you for your quick comment. Yes, normalization and standardization are different. My comment was wrong. This parameter does not provide standardization.

At the same time, I disagree with you. This parameter is not normalization because the applied image vector's norm is not `1.`. (Ref: [scikit-learn's normalize](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html))

Is `画素値のリスケーリング係数` better?

**Comment by MasanoriTakaoka** [comment]
Nice. I totally agree with your idea to avoid ambiguous of normalization or `正規化` in Japanese. Normalization has large meaning like using L1 norm, L2 norm, etc. I'll re-push. Anyway I'm happy to have a discussion with you. Thanks.

**Comment by nzw0301** [comment]
Thank you for your PR and discussion!
